pawn/Resources/Callbacks/OnPlayer/OnPlayerText.pwn

```diff
@@ -8,6 +8,11 @@
  * @param playerid Id of the player who typed the text.
  * @param text The text the player typed.
  */
+ new QuitJokes[] = {
+        "/q",
+        "/quit",
```

Comment:
These lines start with six spaces. They have to start with four spaces.

In addition, how does this even compile? I think it should be QuitJokes[2][]?